object-filter/src/main/java/org/infinispan/objectfilter/impl/predicateindex/RowMatcherEvalContext.java

```diff
@@ -11,7 +11,7 @@ public final class RowMatcherEvalContext extends MatcherEvalContext<RowPropertyH
    private final RowPropertyHelper.RowMetadata rowMetadata;
 
    public RowMatcherEvalContext(Object userContext, Object eventType, Object key, Object instance, RowPropertyHelper.RowMetadata rowMetadata) {
-      super(userContext, eventType, key, instance);
+      super(userContext, eventType, key, instance, null);
```

Comment:
isn't the metadata missing?

Reply:
The `RowMatcherEvalContext` does not support metadata